src/modules/SIGNALPROCESSING/FIELDBUS_PERCENT_TO_WORD_fct.h

```diff
@@ -55,42 +54,34 @@ class FORTE_signalprocessing__FIELDBUS_PERCENT_TO_WORD final : public CFunctionB
   public:
     FORTE_signalprocessing__FIELDBUS_PERCENT_TO_WORD(CStringDictionary::TStringId paInstanceNameId, forte::core::CFBContainer &paContainer);
 
-    CIEC_BOOL var_QI;
     CIEC_REAL var_RI;
 
-    CIEC_BOOL var_QO;
-    CIEC_WORD var_WO;
+    CIEC_WORD var_;
```

Comment:
something is broken here

Reply:
no, i guess it is right. 
`CIEC_WORD var_;`
is the Return Value from a Function, which was exactly the Reason of this PR, to use a Return Value instead of a OUT. 
so perfectly Fine.